plugin/helpers_plugin.go

```diff
@@ -54,3 +54,16 @@ func (p *HelpersImpl) ensureServerVersion(required string) error {
 	}
 	return nil
 }
+
+// GetPluginAssetURL builds a URL to the resource from asset directory.
+func (p *HelpersImpl) GetPluginAssetURL(pluginID, asset string) (*url.URL, error) {
+	if len(pluginID) == 0 || len(asset) == 0 {
+		return nil, errors.New("empty pluginID or directory name provided")
+	}
+	siteURL := *p.API.GetConfig().ServiceSettings.SiteURL
```

Comment:
style nit: let's add a newline before this line, to group things a little more clearly.

Reply:
There's a tiny chance that `SiteURL` is an empty string -- suggest we check for that and also return an error.

Reply:
Thanks for the thoughtful feedback, I'll work on them.